### PR TITLE
Test pneumatics and fix stuff

### PR DESCRIPTION
**Pull request by MeechaPooch**

### CODE REVIEW

File: src/main/java/org/team1540/elmo/RobotContainer.java [hunk 1]
```diff
@@ -58,29 +65,35 @@ public void setupMiscComponents() {
...
+
+        // toggle individual suction when either bumper and X:inner / B:outer / Y:upper are pressed
+        new JoystickButton(driver,XboxController.Button.kX.value).and(eitherBumper).toggleWhenActive(
+            new ResetEjectorAndWaitForeverCommand(innerEjector,true)
+        );
+        new JoystickButton(driver,XboxController.Button.kB.value).and(eitherBumper).toggleWhenActive(
+            new ResetEjectorAndWaitForeverCommand(outerEjector,true)
+        );
+        new JoystickButton(driver,XboxController.Button.kY.value).and(eitherBumper).toggleWhenActive(
+            new ResetEjectorAndWaitForeverCommand(upperEjector,true)
```
Comment (rutmanz) [inline src/main/java/org/team1540/elmo/RobotContainer.java:92]: This control setup is very clever, I assume it's just for testing though? Outside of testing, the pneumatics will only be controlled during auto

File: src/main/java/org/team1540/elmo/subsystems/ejectors/ResetEjectorAndWaitForeverCommand.java [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
         addCommands(
                 parallel(
                         new InstantCommand(()->ejector.setExtended(false)),
-                        new InstantCommand(()->ejector.setSuction(false))
-                        // new InstantCommand(()->ejector.setSuction(true))
+                        new InstantCommand(()->ejector.setSuction(sucking))
                 ),
-                new WaitDashboardVarCommand(0.5,"retract",this)
+                new WaitDashboardVarCommand(0.5,"retract",this),
+                new RunCommand(()->{})
```
Comment (rutmanz) [inline src/main/java/org/team1540/elmo/subsystems/ejectors/ResetEjectorAndWaitForeverCommand.java:17]: You can also force this to never end by overriding isFinished and never returning true, but this way seems just as good

**Review by rutmanz — APPROVED** [review]
Nice work, this all looks really good

File: src/main/java/org/team1540/elmo/RobotContainer.java (hunk 1 above)
Comment (MeechaPooch) [reply src/main/java/org/team1540/elmo/RobotContainer.java:92]: Yeah testing D; Maybe there can be a button to extend the outer suction and pick up vertical standing tubes

File: src/main/java/org/team1540/elmo/subsystems/ejectors/ResetEjectorAndWaitForeverCommand.java (hunk 2 above)
Comment (MeechaPooch) [reply src/main/java/org/team1540/elmo/subsystems/ejectors/ResetEjectorAndWaitForeverCommand.java:17]: That doesnt work for sequential command groups D;

**Pull request merged**